src/util/options.schema.json

```diff
@@ -1433,12 +1452,6 @@
           "type": "boolean",
           "default": false
         },
-        "print_dead_code": {
-          "title": "dbg.print_dead_code",
-          "description": "Print information about dead code",
-          "type": "boolean",
-          "default": false
-        },
```

Comment:
Just as a post-merge note: the renaming of `dbg.print_dead_code` and `dbg.uncalled` will probably require a bunch of renaming in the bench repo as well.

Reply:
True!